core/src/main/scala/com/softwaremill/react/kafka/TopicPartitionOffset.scala

```diff
@@ -0,0 +1,16 @@
+package com.softwaremill.react.kafka
+
+/**
+ * Used as a parameter to [[ReactiveKafkaConsumer]] when consuming from
+ * a specified offset on a designated partition of a Kafka topic.
+ *
+ * @param topic
+ * @param partition
+ * @param offset
+ */
+case class TopicPartitionOffset(topic: String, partition: Int, offset: Long)
+
+object TopicPartitionOffset {
```

Comment:
Yes, the object and its constants can be removed as the constants are no longer referenced.